.travis.yml

```diff
@@ -13,27 +13,33 @@ services:
 
 # Make sure the instances listed below match up with
 # the `platforms` defined in `kitchen.yml`
+# NOTE: To minimise disruption across the whole org, only select five instances
```

Comment:
@myii, perhaps rephrasing like this provides some insight on the reasoning to use when chosing

"...org, please try to select *up to six instances* that add some *meaningful* testing of the formula's behaviour. If possible, try to refrain from the classical "chosing all the instances because I want to test on another/all distro/s" trap: it will just add time to the testing (see the discussion on https://github.com/saltstack-formulas/template-formula/pull/121). 

As an example, the set chosen below covers the most used distros families, systemd and non-systemd and the latest three supported Saltstack versions with python2 and 3."

wdyt?

Reply:
Thanks, have added this comment.